tensorboard/webapp/plugins/npmi/store/npmi_reducers_test.ts

```diff
@@ -472,6 +472,10 @@ describe('npmi_reducers', () => {
         expect(nextState.metricArithmetic).toEqual([
           {kind: ArithmeticKind.METRIC, metric: 'nPMI@test'},
         ]);
+        expect(nextState.sorting).toEqual({
```

Comment:
This is what you missed. Should be `nextState.sort`.

Reply:
Fixed; thanks. I ran `bazel build //tensorboard`, which worked, but
didn’t build the web tests. Everything actually passes now. :-)